api/src/main/java/org/pmiops/workbench/iam/IamServiceImpl.java

```diff
@@ -74,13 +74,10 @@ public List<String> revokeWorkflowRunnerRoleForUsers(
       List<String> petServiceAccountsToRevokePermission = new ArrayList<>();
       for (String userEmail : userEmails) {
         // TODO can we make these requests in parallel?
-        Optional<String> petSaMaybe =
-            getOrCreatePetServiceAccountUsingImpersonation(googleProject, userEmail);
-        if (petSaMaybe.isPresent()) {
-          petServiceAccountsToRevokePermission.add(petSaMaybe.get());
-        } else {
-          petServiceAccountFailures.add(userEmail);
-        }
+        getOrCreatePetServiceAccountUsingImpersonation(googleProject, userEmail)
```

Comment:
not sure if this one is actually better

Reply:
The concision here makes it easier to read.